core/services/chainlink/application.go

```diff
@@ -446,18 +446,20 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 		}
 	}
 
-	// TODO: Make feeds manager compatible with multiple chains
-	// See: https://app.clubhouse.io/chainlinklabs/story/14615/add-ability-to-set-chain-id-in-all-pipeline-tasks-that-interact-with-evm
 	var feedsService feeds.Service
 	if cfg.FeatureFeedsManager() {
 		feedsORM := feeds.NewORM(db, opts.Logger, cfg)
-		chain, err := chains.EVM.Default()
```

Comment:
We don't need chain specific config values anymore, so this can be removed in favour of passing in the general config.